src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+// Actions
+export const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+export const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+// initial state
+const bookState = [];
+
+// Action Creators
+export const addBook = () => ({
+  type: ADD_BOOK,
+});
+
+export const removeBook = () => ({
+  type: REMOVE_BOOK,
+});
+
+// Reducer
+
+const bookReducer = (state = bookState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+      ];
+    case REMOVE_BOOK:
+      return [
+        ...state,
+      ];
```

Comment:
- [ ] Kindly make sure when in ADD_BOOK action to add the new book to the state and also for REMOVE_BOOK to remove the book from the state by using the filter method.

Reply:
@omarsalem7 am I to add books to the books component?